Classes/ViewHelpers/IconViewHelper.php

```diff
@@ -59,7 +68,13 @@ public function initializeArguments(): void
     {
         parent::initializeArguments();
         $this->registerArgument('icon', 'string', 'The icon name', true);
-        $this->registerArgument('provider', 'string', 'PageTS did of the used IconProvider', false);
+        $this->registerArgument(
+            'markup',
+            'string',
+            'Markup of the',
```

Comment:
Is this supposed to be cut off mid sentence? I don't remember how exactly TYPO3 is displaying those, so maybe it's alright, in which case please disregard this.

Reply:
You're right, I missed it.